### ISSUE TITLE
Coverpage: lack of focus outline and colour change of sufficient contrast

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#169]
### Description

Many keyboard focusable elements on the cover page lack of focus styling or sufficient focus contrast. In particular:
* Header logo
* hamburger menu (mobile layout)
* search field
* top navigation items i.e.  home, read, admin, sign-out (desktop layout)
* Read Book button (colour inverts on focus, but lacks focus border)
* Table of Contents section header
* Table of Contents chapters
* Hide All Contents button
* Links in footers

### Steps to Reproduce

1. Go to the cover page of a book.
2. Using the keyboard, press the Tab key repeatedly to move focus through the document.
3. Notice the general lack of focus styling or insufficient contrast.

**Expected behavior:**
* Focus colour change and outline that meets WCAG 2.0 AA colour contrast guidelines
* On elements (like buttons) where adding a border or outline just makes the button look bigger, use the[ double box-shadow technique](https://css-tricks.com/snippets/css/multiple-borders/) to create an illusion of a gap between borders.

**Comment by mcgratay** [comment]
These issues should all be resolved in the Edge, Chrome, and Firefox browsers. We have another issue open to tackle the Safari browser: https://github.com/pressbooks/pressbooks/issues/1883
